### ISSUE TITLE
Forum don't display own posts when referring with @

### ISSUE THREAD

**Issue by hoergen** [friendica/friendica#5225]
### Expected behavior
Opening the forum should display own posts that are referred with @ 

### Actual behavior
It only shows posts that are referred with the bang notation ! but not with the @ notation

### Steps to reproduce the problem
Write a post an refer it with a @<name of forum>

Tested it on my local node with a local forum

### Friendica version you encountered the problem
Friendica develop / git / php7.0 / 5.5.5-10.0.32-MariaDB-0+deb8u1

**Comment by annando** [comment]
It is not really a bug, but simply based upon the data model - and the reason why we introduced the ```!```.

**Comment by hoergen** [comment]
Hmm but when I look at the forum from the forum user itself I can see that post.
If I look as non-logied in user I can see that post.
Only the user that wrote that post can't see is own post.

soooo it looks like a bug ;)

I understood the difference between @ and ! is like "shared between contacts and forum" and "exclusively forum"

**Comment by annando** [comment]
We can only display posts under the user's wall **or** the forum page - not both. Posts with ```@``` are put on the user's wall, since we define that these posts are *mentioning* the forum, but aren't primarily forum posts.

**Comment by annando** [comment]
The data model won't allow us to show these posts under your profile **and** on the local forum page. I do understand your point. But I don't know how to realise this at the moment.

**Comment by MrPetovan** [comment]
I don't agree with @hoergen, when you mention someone with `@`, you don't expect to see your post on their timeline. Why would it be different with forum accounts when there's another way of posting to the forum?

**Comment by annando** [comment]
The difference is that you see such mentioned forum posts on the remote forum profile - but not the locale one. So I do understand the point.

**Comment by MrPetovan** [comment]
Ah, then it's different, of course. If you can't show it locally, then it probably should be hidden on the remote profile as well for consistency.

**Comment by hoergen** [comment]
@annando I know that is why I suggested "Feature Request"

@MrPetovan Hiding it is really no good idea, that will confuse people even more. For consistency we need to keep it in mind to solve it one day like that we can see it in our own timeline AND in the forum.
